src/data_ingestion/reading_data.py

```diff
@@ -0,0 +1,9 @@
+def load_csv_into_spark_data_frame(spark, data_hes_path):
+    '''
+    A function to load the data from CSV into a spark dataframe using path defined
+    '''
+    df_hes_data = (spark
+        .read
+        .csv(data_hes_path, header=True)
+    )
```

Comment:
Tiny format change that looks better
```suggestion
    df_hes_data = (
        spark
        .read
        .csv(data_hes_path, header=True)
    )
```

Reply:
I would instead suggest like this. It's not too long to be pep-8 compliant!
```suggestion
    df_hes_data = spark.read .csv(data_hes_path, header=True)
```